src/render/shaders/tilemap_fragment.wgsl

```diff
@@ -1,8 +1,11 @@
-#import bevy_ecs_tilemap::common
```

Comment:
We need `tilemap_data` too for the `atlas` feature.

Reply:
Ah good catch, done!